pkg/controller/installation/products/amqstreams/reconciler.go

```diff
@@ -169,6 +172,17 @@ checkPodStatus:
 		}
 	}
 
+	kafka := &kafkav1.Kafka{}
+	err = client.Get(ctx, pkgclient.ObjectKey{Name: clusterName, Namespace: r.Config.GetNamespace()}, kafka)
+	if err != nil {
+		return v1alpha1.PhaseFailed, errors.Wrap(err, "failed to get kafka custom resource")
+	}
+
+	if string(r.Config.GetProductVersion()) != kafka.Spec.Kafka.Version {
```

Comment:
This is an interesting one same for AMQ Online. It is using AMQ x but is called AMQ Online 1.1.1 this feels like it should be the version of AMQ Streams not kafka?

Reply:
This was the only version number I could find for AMQ Streams, it's the right version though isn't it? Amq Streams is 1.1.x? As for AMQ Online, I couldn't find that version dynamically, so I have hard-coded it.